CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
     - Bugfixes:
         - Add ID attributes to change password form inputs.
         - Fix link deactivation for privacy policy link on privacy policy page. #3704
+    - Admin improvements:
+        - 'Staff' checkbox on `/admin/users` & `/admin/users/add` now permanently checked; submitting the form for a new user guarantees they are assigned to a body.
```

Comment:
This file is generally wrapped to standard length, and tries to fit the change in one line if possible - perhaps something like "Admin add user form now always creates staff" or similar?